content/en/project/BHS_Project_dMRI/index.md

```diff
@@ -0,0 +1,137 @@
+2. [PRIME](http://fcon_1000.projects.nitrc.org/indi/PRIMEdownloads.html): used this dataset to reconstruct diffusion Images. This also can be downloaded [here](https://drive.google.com/file/d/1zgxynxjUCETBC6MAl4rfh0sL0WhFtKA9/view?usp=sharing) directly. 
+3. Since during preprocessing, we used epi data with two opposite phase-encoding directions to correct distortions, other data can also be used as the source data if it meets this requirement.
+4. Data used for analysis is generated from image data after preprocessing. The final data can be found in this [github project folder](https://github.com/brainhack-school2020/BHS_Project_dMRI/blob/master/brain_water.csv).
+
+### Deliverables
+
+At the end of this project, we will have: Jupyter notebook will be developed, allowing diffusion MR reconstruction and data analyzation. Weekly Deliverables are as following:
+ 
+ - Week 1: [Assignment 2](https://github.com/zhangerjun/Zhang-EJ-QLS612)
+ - Week 2: [README file](https://github.com/brainhack-school2020/BHS_Project_dMRI/blob/master/README.md)
+ - Week 3: [Data visualization](https://github.com/brainhack-school2020/BHS_Project_dMRI/tree/master/Visualization)
+ - Week 4: [Project report](https://github.com/brainhack-school2020/BHS_Project_dMRI) and [project prosentation](https://github.com/brainhack-school2020/BHS_Project_dMRI/blob/master/BHS_dMRI_Project.pdf)
+
+## Results
+
+### Progress overview
+
+Since May 11, this project goes well and three goals of this project have been reached. This is just to let me be familiar with steps and tools of diffusion MRI processing. Next, I will combine different processing methods into this project. Specifically, I would like to look deep into DTI model fitting and try to replace it by a new model created by myself.
+
+### Tools I learned during this project
+
+1. Python
+	* FSL, Nipype.interfaces, DIPY
+	* matplotlib, plotly, ipywidgets
+	* Function definition and calling
+2. Markdown
+	* Format this README file
+	* Insert figures and animation
+	* LaTex to edit mathematical equations
+3. Github
+	* Version control
+	* Project organization
+4. BIDS
+5. Jupter notebook
+6. Found and learnt to use some small but useful tools
+	* [Peek](https://github.com/phw/peek) for GIF making
+	* [Gedit](https://wiki.gnome.org/Apps/Gedit) to edit code or markdown
+7. Medical file format (NII) and data structure (BIDS)
+8. Statistical method and machine learning
+
+### Results
+
+#### Data Visualization (see [gif figure](https://github.com/brainhack-school2020/BHS_Project_dMRI/tree/master/Visualization) below):
+  - 3D volume slices image 
+  - Interactive widgets use to show preprocess results 
+  - Interactive widgets use to show reconstruction results 
+<div align="center">
+	<img src="https://github.com/brainhack-school2020/BHS_Project_dMRI/blob/master/Visualization/VolumeSlicesImg.gif" width="300" alt="1" title="3D volume slices image">
```

Comment:
this link is broken. To render on the website, you need to add the file to the project folder, and then link like `VolumeSlicesImg.gif`